annotations-impl/src/main/java/org/ocpsoft/rewrite/annotation/scan/WebClassesFinder.java

```diff
@@ -174,13 +174,13 @@ private void handleClassEntry(URL entryUrl, String entryName, ClassVisitor visit
              */
             try
             {
-               classFileStream = entryUrl.openStream();
+              classFileStream = servletContext.getResourceAsStream(entryName);
             }
-            catch (IOException e)
+            catch (Exception e)
```

Comment:
No, that's ok. I will merge the pull request and do this afterwards. I just wanted to learn about the reason you did it that way... :)